roles/keystone/templates/etc/shibboleth/sp-cert.pem

```diff
@@ -0,0 +1 @@
+{{ ssl.crt }}
```

Comment:
Are we going to use the same key and cert for SAML as well? I think we should create a different one for SAML